cli/tests/unit/signal_test.ts

```
@@ -203,3 +203,19 @@ Deno.test(
     });
   },
 );
+
+Deno.test(
+  {
+    ignore: Deno.build.os === "windows",
+    permissions: { run: true },
+  },
+  function signalForbiddenSignalTest() {
+    assertThrows(
+      () => {
+        Deno.addSignalListener("SIGKILL", () => {});
```

Could you test all signals that should throw this error?